js/common.js

```diff
@@ -0,0 +1,25 @@
+function copyCodePanel(id) {
```

Comment:
We should be using `use strict`. Probably could be added to existing javascript files as well.

Reply:
Given that they're switching to typescript, this'll be taken care of at that point (with the `--alwaysStrict` option)